gradle/releaser.gradle

```diff
@@ -125,10 +125,12 @@ static def repoKeyFor(p) {
 			(~/^\d+\.\d+\.\d+$/) : "libs-release-local",
 			(~/^\d+\.\d+\.\d+-SNAPSHOT$/) : "libs-snapshot-local",
 			(~/^\d+\.\d+\.\d+\.M\d+$/) : "libs-milestone-local",
+			(~/^\d+\.\d+\.\d+\.RC\d+$/) : "libs-milestone-local",
```

Comment:
actually, I just noticed the regexp is wrong for new scheme: milestones and release candidates have the `-M` and `-RC` prefixes, using a dash not a dot. Examples: `3.4.0-M1`, `3.4.3-RC9`.